doc/modules/ROOT/partials/algorithms/triangle-count/specific-configuration-stream.adoc

```diff
@@ -0,0 +1,2 @@
+| maxDegree            | Integer | 2^63^ - 1 | yes      | If a node has a degree higher than this it will not be considered by the algorithm. The triangle count for these nodes will be `-1`.
+| minCommunitySize     | Integer | 0         | yes      | Only nodes inside communities larger or equal the given value are returned.
```

Comment:
quick question, should we have 0 or 1 as the minimum value?

Reply:
I would say 0 should be the minimum value.
Also thinking about TriangleCount more, it is not a component/communityId. So we cannot use the CommunityProcCompanion logic.
Each triangleCount is independent and just because two nodes both have `4` triangles does not mean they belong together.
We could have simple value based filter which only returns nodes with more than X triangles. But that has comparably little value IMO.

Hence, I would actually not add `minTriangles` to TriangleCount add all here.
Open for other arguments of course :)

Reply:
I would say not adding the parameter would be the better thing to do.

Reply:
Sorry for letting you do this extra work

Reply:
no worries, I am actually happy that you shared your thoughts on this, because I learned something.